scripts/e2etests/VSUtils.ps1

```diff
@@ -2,11 +2,44 @@ $VSInstallerProcessName = "VSIXInstaller"
 
 . "$PSScriptRoot\Utils.ps1"
 
+function Get-LatestVSInstance
```

Comment:
This name almost got me. Worth including the fact that it looks for the appropriate version baed on our config props?

SO, `Get-LatestCompatibleVSInstance` maybe?

Reply:
I agree, I started off writing it to get the latest version, but then changed it to as you described. I decided to go a different way, I kept the function the same, removed the config.props reading, and added a `VersionRange` parameter. I added a new `Get-VisualStudioVersionRangeFromConfig` function, and changed the 2 places that call `Get-LatestVSInstance` to call both methods. Maybe it's a little extra complicated, but they're useful utility functions, and make it a little easier to debug these scripts, I think.